scripts/wpilib_dependencies.py

```diff
@@ -28,7 +28,7 @@ def _halsim_dependency(maven_dep, artifact_name):
 def get_wpilib_dependencies():
 
     MAVEN_URL = "https://frcmaven.wpi.edu/release"
-    VERSIONS = ["2021.3.1", "2022.1.1-beta-1"]
+    VERSIONS = ["2021.3.1", "2022.1.1-beta-2"]
```

Comment:
I would say either purge the beta1 deps.bzl file, or keep it as a supported version here. If it isn't in here, the user can't select it, and thus its dead code.